packages/transition-common/src/services/simulation/simulationMethod/index.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2025, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+
+import type { SimulationAlgorithmDescriptor } from '../SimulationAlgorithm';
+import { AccessMapSimulationDescriptor, type AccessMapSimulationOptions } from './AccessibilityMapSimulationMethod';
+import { OdTripSimulationDescriptor, type OdTripSimulationOptions } from './OdTripSimulationMethod';
+
+export type { AccessMapSimulationOptions } from './AccessibilityMapSimulationMethod';
+export type { OdTripSimulationOptions } from './OdTripSimulationMethod';
+
+// Define the registry of all available simulation methods
+export interface SimulationMethodRegistry {
```

Comment:
@kaligrafy @greenscientist question: in #1553 @coderabbitai indirectly suggested to call these `evaluationMethods` instead of `simulationMethods`. Would you agree? I think it makes sense as it is used to rate/evaluate a given transit network scenario. And `Simulation` refers to the high-level concept that uses an algorithm to suggest optimal scenario within certain parameters. Since we're reworking the architecture around the simulation, we might as well work on the correct naming right away, to avoid confusion.